provider/docker.go

```diff
@@ -225,15 +225,22 @@ func (provider *Docker) getLabel(container docker.Container, label string) (stri
 }
 
 func (provider *Docker) getLabels(container docker.Container, labels []string) (map[string]string, error) {
+	var globalErr error
 	foundLabels := map[string]string{}
 	for _, label := range labels {
 		foundLabel, err := provider.getLabel(container, label)
+		// Error out only if one of them is defined.
 		if err != nil {
-			return nil, errors.New("Label not found: " + label)
+			globalErr = errors.New("Label not found: " + label)
+			continue
 		}
 		foundLabels[label] = foundLabel
+
+	}
+	if len(foundLabels) == 0 {
```

Comment:
@emilevauge Pretty sure yes :stuck_out_tongue_closed_eyes: I want to error out only if at least one of the given labels have been found. The use case is the following :

1. I want traefik to handle a container I started without *any* labels, using default stuff (using the name for the frontend rules/values, etc..)
2. I want traefik to force me to define `traefik.frontend.value` and `traefik.frontend.rules` together and filter out the container if only one of them is defined.

As it's currently implemented (on master), it filters all containers that didn't define both of these labels… which breaks the default behavior.

But on that note, the name of this method should be revised as it's not a simple `getLabel`.